docs/it/week11/11-2.md

```diff
@@ -0,0 +1,559 @@
+
+<!-- $$
+\begin{equation}
+l_n =
+\left\{
+             \begin{array}{lr}
+             x_n, &\quad y_n=1,  \\
+             \max\{0,\Delta-x_n\}, &\quad y_n=-1  \\
+             \end{array}
+\right.
+\end{equation}
+$$
+
+Hinge embedding loss used for semi-supervised learning by measuring whether two inputs are similar or dissimilar. It pulls together things that are similar and pushes away things are dissimilar. The $y$ variable indicates whether the pair of scores need to go in a certain direction. Using a hinge loss, the score is positive if $y$ is 1 and some margin $\Delta$ if $y$ is -1. -->
+
+### Perdita di coseno ad immersione - `nn.CosineEmbeddingLoss()`
+
+<!-- ### Cosine Embedding Loss - `nn.CosineEmbeddingLoss()` -->
+
+$$
+l_n =
+  \left\{
+             \begin{array}{lr}
+             1-\cos(x_1,x_2), & \quad y=1,  \\
+             \max(0,\cos(x_1,x_2)-\text{margine}), & \quad y=-1
+             \end{array}
+   \right.
+$$
+
+Questa funzione di perdita misura la somiglianza fra due input, utilizzando la distanza del coseno di similitudine, e viene tipicamente utilizzata per apprendere immersioni non-lineari o per l'apprendimento semi-supervisionato.
+
+* Definito in maniera alternativa, 1 meno il coseno dell'angolo fra due vettori è praticamente la distanza Euclidea normalizzata.
+* Il vantaggio <!-- svantaggio? --> di questa funzione è che quando si hanno due vettori e li si vuole rendere molto distanti fra di loro, lo si può ottenere semplicemente rendendo i vettori molto lunghi. Ovviamente questo non è ottimale, perché non vogliamo aumentare la lunghezza dei vettori ma ruotarli nella direzione giusta. Per questo, si normalizzano i vettori e si calcola la distanza Euclidea fra di loro.
+* Per le coppie di vettori "positive" (appartenenti alla stessa categoria), questa perdita tenta di allineare i vettori fra di loro il più possibile. Per le coppie "negative" (appartenenti a categorie diverse), la perdita rende il coseno minore di un margine, che deve avere un valore basso ma positivo.
+* In uno spazio ad alta dimensionalità, abbiamo una concentrazione alta di punti <!--?--> attorno all'"equatore" della sfera. Dopo aver normalizzato i vettori, tutti i punti si trovano distribuiti sulla sfera. Desideriamo che i punti semanticamente simili siano vicini. Gli esempi dissimili, invece, dovrebbero essere ortogonali. Non vogliamo che siano in direzione opposta perché vi è un solo punto al polo opposto <!--?-->. Dato che sull'"equatore" abbiamo una grande quantità di spazio, vorremmo avere un margine dal valore basso cosí da poter sfruttare tutta l'area in questa regione.
```

Comment:
Si può tradurre "there is a lot of **area** near the the equator of the sphere" come "**concentrazione di punti** attorno all'equatore della sfera"? o meglio lasciare "**area**"